lib/cn/csched_sp3.c

```diff
@@ -1292,7 +1156,7 @@ sp3_process_workitem(struct sp3 *sp, struct cn_compaction_work *w)
     sp3_dirty_node_locked(sp, tn);
     rmlock_runlock(lock);
 
-    if (w->cw_debug & CW_DEBUG_PROGRESS)
+    if (w->cw_debug & (CW_DEBUG_PROGRESS|CW_DEBUG_FINAL))
```

Comment:
Spaces around unary OR

Reply:
I believe that's a binary op, but yes please :-)